packages/react/src/primitives/SearchField/SearchField.tsx

```diff
@@ -13,6 +13,7 @@ const SearchFieldPrimitive: Primitive<SearchFieldProps, 'input'> = (
   {
     autoComplete = 'off',
     className,
+    isDisabled,
```

Comment:
pulling out isDisabled so that it can be passed to both the input and search button